libutils/buffer.c

```diff
@@ -427,38 +427,6 @@ int BufferPrintf(Buffer *buffer, const char *format, ...)
     return printed;
 }
 
-int BufferVPrintf(Buffer *buffer, const char *format, va_list ap)
```

Comment:
OK, I will remove this commit, leaving the function there.

The concern is that it takes format as a parameter, which *could* be a security risk if the caller allows an attacker to inject data there. I added a comment instead.